.github/workflows/packages-build-manager.yml

```
@@ -0,0 +1,153 @@
+          Build package with release format.
+          Checksum will be also generated.
+        type: boolean
+        required: false
+      debug:
+        description: |
+          Build the binaries with debug symbols.
+          By default: false
+        type: boolean
+        required: false
+      checksum:
+        description: Generate package checksum.
+        type: boolean
+        required: false
+
+  workflow_call:
+    inputs:
+      docker_image_tag:
+        required: false
+        default: 'auto'
+        type: string
+      architecture:
+        type: string
+        required: true
+      system:
+        type: string
+        required: true
+      os_name:
+        default: ""
+        type: string
+        required: false
+      revision:
+        default: "0"
+        type: string
+        required: false
+      is_stage:
+        type: boolean
+        required: false
+      debug:
+        type: boolean
+        required: false
+      checksum:
+        type: boolean
+        required: false
+
+jobs:
+  Build-manager-packages:
+    runs-on: ubuntu-latest
+    name: Build ${{ inputs.system }} wazuh-manager on ${{ inputs.architecture }}
+
+    steps:
+      - name: Cancel previous runs
+        uses: fkirc/skip-duplicate-actions@master
+        with:
+          cancel_others: 'true'
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          skip_after_successful_duplicate: 'false'
+
+      - uses: actions/checkout@v4
+
+      - name: Set tag and container name
+        run: |
+          VERSION=$(sed 's/^v\([0-9]*\.[0-9]*\.[0-9]*\)/\1/' $GITHUB_WORKSPACE/src/VERSION)
+          if [ "${{ inputs.docker_image_tag }}" == "auto" ]; then echo "TAG=$VERSION" >> $GITHUB_ENV;
+          elif [ "${{ inputs.docker_image_tag }}" == "developer" ]; then echo "TAG=${{ github.ref_name }}" >> $GITHUB_ENV;
+          else echo "TAG=${{ inputs.docker_image_tag }}" >> $GITHUB_ENV; fi
+          echo "CONTAINER_NAME=pkg_${{ inputs.system }}_manager_builder_amd64" >> $GITHUB_ENV
+
+      - name: Download docker image for package building
+        run: |
+          bash $GITHUB_WORKSPACE/.github/actions/ghcr-pull-and-push/pull_image_from_ghcr.sh ${{ secrets.GITHUB_TOKEN }} ${{ github.actor}} $CONTAINER_NAME ${{ env.TAG }}
+
+      - name: Build ${{ inputs.system }} wazuh-manager on ${{ inputs.architecture }}
+        working-directory: packages
+        run: |
+          REVISION=${{ inputs.revision }}
+          FLAGS="-t manager -s /tmp --dont-build-docker -j $(nproc) "
+          if [ -z "$REVISION" ]; then FLAGS+="-r 0 "; else FLAGS+="-r $REVISION "; fi
+          if [ "${{ inputs.is_stage }}" == "true" ]; then FLAGS+="--is_stage "; fi
+          if [ "${{ inputs.checksum }}" == "true" ]; then FLAGS+="--checksum "; fi
+          if [ "${{ inputs.debug}}" == "true" ]; then FLAGS+="--debug "; fi
+          echo $FLAGS
+          bash generate_package.sh -a amd64 --tag ${{ env.TAG }} --system ${{ inputs.system }} $FLAGS
```

Why is it only for amd64?
```suggestion
          bash generate_package.sh -a ${{ inputs.architecture }} --tag ${{ env.TAG }} --system ${{ inputs.system }} $FLAGS
```

---

This comment should be on the code.